packages/node_modules/@ciscospark/react-component-md-text-input/src/index.js

```diff
@@ -0,0 +1,74 @@
+/* eslint no-underscore-dangle: 0  */
+
+import React from 'react';
+import {Input} from '@momentum-ui/react';
+import * as AdaptiveCard from 'adaptivecards';
+import classnames from 'classnames';
+import {renderCardElement} from '@ciscospark/react-component-utils';
+
+/**
+ * Replaces Text Input of adaptive Card with MD(Momentum Design) Input Component
+ * @returns {HTMLElement}
+ */
+class MDTextInput extends AdaptiveCard.TextInput {
+  constructor(addChildNode) {
+    super();
+    this.addChildNode = addChildNode;
+  }
+
+  /**
+   * returns the assigned value to internalRender() method
+   * @returns {string}
+   */
+  get value() {
+    return this._value;
+  }
+
+  /**
+   * assigns the value when the data is sent in the JSON request
+   * @param {string} newValue
+   */
+  set value(newValue) {
+    this._value = newValue;
+  }
+
+  /**
+   * renders momentum design `<Input />` Component
+   * @returns {HTMLElement}
+   */
+  internalRender() {
+    this.value = this.defaultValue || '';
+
+    // Input style of Adaptive card supports Text, email, phone and url
+    // Momentum Design only has text and email so rendering phone and url as text
+    const inputStyles = {
+      0: 'Text',
```

Comment:
why is this all `Text` and `Text` and `Text`?

Reply:
shouldn't be `Text`, `Email`, `phone` and `url`? also if this had not been noticed, it means that there is no unit test to test this functionality.

Reply:
Adaptive cards are using momentum UI which only has text and email as allowed values Hence this was decided to render phone and URL as text. This explanation is already part of my PR to avoid confusion